linera-client/src/client_options.rs

```diff
@@ -432,6 +432,10 @@ pub enum ClientCommand {
         address: String,
         /// The chain to query. If omitted, query the default chain of the wallet.
         chain_id: Option<ChainId>,
+        /// The public key of the validator. If given, the signature of the chain query
+        /// info will be checked.
+        #[arg(long)]
+        name: Option<ValidatorName>,
```

Comment:
IMHO it's confusing to call a _public key_ a _name_.

Reply:
That's fair. I created: https://github.com/linera-io/linera-protocol/issues/2820